internal/lib/container_server.go

```diff
@@ -31,6 +31,9 @@ import (
 // `io.container.manager`.
 const ContainerManagerCRIO = "cri-o"
 
+// UsernsMode is the user namespace mode to use
+const UsernsModeAnnotation = "io.kubernetes.cri-o.userns-mode"
```

Comment:
This should go into `annotations` package. I am still not sure at what point we moved that to libpod since all the annotations are crio specific in that package.

Reply:
We can handle this as follow-up but we should leave a TODO here.

Reply:
added